src/main/kotlin/racingcar/domain/racing/AttemptCount.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar.domain.racing
+
+import racingcar.exception.ErrorMessage
+
+@JvmInline
+value class AttemptCount private constructor(val value: Int) {
+
+    companion object {
+        fun from(input: String): AttemptCount {
+            require(input.isNotBlank()) { ErrorMessage.ATTEMPT_COUNT_BLANK.message }
+            val n = input.toIntOrNull() ?: throw IllegalArgumentException(ErrorMessage.ATTEMPT_COUNT_NOT_POSITIVE.message)
```

Comment:
이 부분에서 requireNotNull을 활용해서 작성할 수도 있을 것 같습니다!
```suggestion
            val n = requireNotNull(input.toIntOrNull()) { ErrorMessage.ATTEMPT_COUNT_NOT_POSITIVE.message }
```

Reply:
이 부분만 `require`을 사용하지 못한 게 아쉬웠는데 방법이 있었군요! 감사합니다!